package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "treemapper",
-  "version": "1.0.6",
+  "version": "1.0.8",
```

Comment:
I think we do not have to increase the version to 1.0.8 - using 1.0.7 is still fine. Just the build number has to be higher than in the current release branch as it is already used up at AppStore Connect.

Reply:
ok will make the change